### DISCUSSION TITLE
Change Text for Login Link

### DISCUSSION THREAD

**Discussion by joshgk00** [vanjarosoftware/Vanjaro.Platform#1967] [Q&A] [answered]
Is there any way to change the text for the login text to say something other than My Account? I didn't see anything in the settings for the block. 

![image](https://user-images.githubusercontent.com/13650025/145592554-f0af5b73-3d3d-4952-b6c1-8b6c4c59d763.png)

**Comment by meetmandeep** [comment, marked as answer]
That should be localized. You can change from languages
